entrypoint.sh

```diff
@@ -52,6 +52,9 @@ else
     DOC_TARGET_DIR=.
 fi
 
+# Work around the "fatal: detected dubious ownership" issue
+git config --global --add safe.directory '*'
```

Comment:
suggestion: can you add a link to where the workaround originated?

Reply:
Added a ref to the GitHub comment on the larger upstream discussion.